### PR TITLE
adding batch insert

### PR DESCRIPTION
**Pull request by newbie456**

### CODE REVIEW

File: livemaker/cli.py [hunk 1]
```diff
@@ -791,6 +764,51 @@ def edit(lsb_file, line_number):
 def main():
     pass
 
+def insert_LNS(encoding, lsb_file, script_file, line_number, no_backup):
```
Comment (pmrowla) [inline livemaker/cli.py]: PEP8: rename this to `insert_lns()`

File: livemaker/cli.py [hunk 2]
```diff
@@ -305,16 +305,19 @@ def extract(encoding, output_dir, input_file):
...
-            if name:
-                name = '{}-{}.lns'.format(lsb_path.stem, name)
-            if not name:
-                name = '{}-line{}.lns'.format(lsb_path.stem, line)
-            output_path = output_dir.joinpath(name)
-            dec = LNSDecompiler()
-            with open(output_path, 'w', encoding=encoding) as f:
-                f.write(dec.decompile(scenario))
-            print('  wrote {}'.format(output_path))
+        lsbRefFileName = '{}.lsbref'.format(lsb_path.stem)
```
Comment (pmrowla) [inline livemaker/cli.py]: pep8: rename this to `lsb_ref_filename`

File: livemaker/cli.py [hunk 3]
```diff
@@ -305,16 +305,19 @@ def extract(encoding, output_dir, input_file):
...
+        with open(output_dir.joinpath(lsbRefFileName), 'w', encoding=encoding) as lsbRefFile:
```
Comment (pmrowla) [inline livemaker/cli.py]: pep8

File: livemaker/cli.py [hunk 4]
```diff
@@ -343,6 +343,44 @@ def insert(encoding, lsb_file, script_file, line_number, no_backup):
...
+    """
+    script_dir = Path(script_dir)
+    if not script_dir.exists:
+        print('Input directory does not exist')
+        return
+    if not no_backup:
+        print('Backing up original LSB.')
+        shutil.copyfile(str(lsb_file), '{}.bak'.format(str(lsb_file)))
+    lsb_path = Path(lsb_file)
+    lsbRefFileName = '{}.lsbref'.format(lsb_path.stem)
```
Comment (pmrowla) [inline livemaker/cli.py]: pep8

**Comment by newbie456** [comment]
i hope it is ok now

**Pull request merged**